### PR TITLE
Allow user with multiple roles, add UserManager role

### PR DESCRIPTION
**Pull request by rnijveld**
This implements the first part of #82 I added the ability to add multiple roles specifically to allow creating a kind of superuser account for testing purposes.

### CODE REVIEW

File: openadr-vtn/src/jwt.rs [hunk 1]
```diff
@@ -19,20 +19,83 @@ pub struct JwtManager {
...
+                if let AuthRole::VEN(id) = role {
+                    Some(id.clone())
+                } else {
+                    None
+                }
+            })
+            .collect()
+    }
+
+    pub fn business_ids(&self) -> (Vec<String>, bool) {
```
Comment (tdittr) [inline openadr-vtn/src/jwt.rs:66]: Would this be better as an `enum Foo {Any, Specific(Vec<String>)}`?

File: openadr-vtn/src/jwt.rs [hunk 2]
```diff
@@ -19,20 +19,83 @@ pub struct JwtManager {
...
     exp: usize,
     nbf: usize,
     pub sub: String,
-    pub role: AuthRole,
-    pub ven: Option<String>,
+    pub roles: Vec<AuthRole>,
+}
+
+impl Claims {
+    pub fn ven_ids(&self) -> Vec<String> {
```
Comment (tdittr) [inline openadr-vtn/src/jwt.rs:53]: Could this be a new-type instead of `String`?

**Review by tdittr — CHANGES_REQUESTED** [review]

File: openadr-vtn/src/jwt.rs (hunk 2 above)
Comment (rnijveld) [reply openadr-vtn/src/jwt.rs:53]: Probably will be, but I wanted to keep it smaller.

File: openadr-vtn/src/jwt.rs (hunk 1 above)
Comment (rnijveld) [reply openadr-vtn/src/jwt.rs:66]: Good idea! Will change!

**Review by tdittr — APPROVED** [review]

**Pull request merged**